yarn.lock

```diff
@@ -6367,6 +6367,7 @@ __metadata:
     prettier: "npm:3.2.5"
     redis: "npm:^4.7.0"
     typescript: "npm:5.4.3"
+    ulid: "npm:^2.3.0"
     uuid: "npm:^11.0.5"
```

Comment:
is `uuid` still needed?

Reply:
We are using it in table storage adapter